content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -4,86 +4,104 @@ reviewers:
 
-- [Upgrading kubeadm cluster from 1.17 to 1.18](https://v1-18.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
-- [Upgrading kubeadm cluster from 1.16 to 1.17](https://v1-17.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
-- [Upgrading kubeadm cluster from 1.15 to 1.16](https://v1-16.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
-- [Upgrading kubeadm cluster from 1.14 to 1.15](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15/)
-- [Upgrading kubeadm cluster from 1.13 to 1.14](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14/)
+- From {{< skew latestVersionAddMinor -1 >}} to {{< skew latestVersion >}}
+- From {{< skew latestVersionAddMinor -2 >}} to {{< skew latestVersionAddMinor -1 >}}
+- From {{< skew latestVersionAddMinor -3 >}} to {{< skew latestVersionAddMinor -2 >}}
+- From {{< skew latestVersionAddMinor -4 >}} to {{< skew latestVersionAddMinor -3 >}}
+- From {{< skew latestVersionAddMinor -5 >}} to {{< skew latestVersionAddMinor -4 >}}
 
 The upgrade workflow at high level is the following:
 
-1. Upgrade the primary control plane node.
+1. Upgrade a primary control plane node.
 1. Upgrade additional control plane nodes.
 1. Upgrade worker nodes.
 
 ## {{% heading "prerequisites" %}}
 
-- You need to have a kubeadm Kubernetes cluster running version 1.18.0 or later.
-- [Swap must be disabled](https://serverfault.com/questions/684771/best-way-to-disable-swap-in-linux).
-- The cluster should use a static control plane and etcd pods or external etcd.
 - Make sure you read the [release notes]({{< latest-release-notes >}}) carefully.
+- The cluster should use a static control plane and etcd pods or external etcd.
 - Make sure to back up any important components, such as app-level state stored in a database.
   `kubeadm upgrade` does not touch your workloads, only components internal to Kubernetes, but backups are always a best practice.
+- [Swap must be disabled](https://serverfault.com/questions/684771/best-way-to-disable-swap-in-linux).
 
 ### Additional information
 
+- [Draining nodes](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) before kubelet MINOR version
+  upgrades is required. In the case of control-plane nodes, they could be running CoreDNS Pods or other critical workloads.
 - All containers are restarted after upgrade, because the container spec hash value is changed.
-- You only can upgrade from one MINOR version to the next MINOR version,
-  or between PATCH versions of the same MINOR. That is, you cannot skip MINOR versions when you upgrade.
-  For example, you can upgrade from 1.y to 1.y+1, but not from 1.y to 1.y+2.
 
 <!-- steps -->
 
 ## Determine which version to upgrade to
 
-Find the latest stable 1.19 version:
+Pick a target version from this list to update the examples bellow:
```

Comment:
Typo: “bellow”.

Reply:
:+1:  will update once we decide on:
https://github.com/kubernetes/website/pull/24908#issuecomment-730515093